test/unit/org/apache/cassandra/utils/AbstractTypeGenerators.java

```diff
@@ -64,6 +99,30 @@
     private static final Gen<Integer> VERY_SMALL_POSITIVE_SIZE_GEN = SourceDSL.integers().between(1, 3);
     private static final Gen<Boolean> BOOLEAN_GEN = SourceDSL.booleans().all();
 
+    public static final Map<Class<? extends AbstractType<?>>, String> UNSUPPORTED = ImmutableMap.<Class<? extends AbstractType<?>>, String>builder()
```

Comment:
Nit: we might want to annotate the entire class with `@SuppressWarnings({"unchecked", "rawtypes"})`.